src/logic/safe/transactions/gas.ts

```diff
@@ -29,65 +24,67 @@ const estimateDataGasCosts = (data: string): number => {
   return data.match(/.{2}/g)?.reduce(reducer, 0)
 }
 
-export const estimateTxGasCosts = async (
+// https://docs.gnosis.io/safe/docs/docs5/#pre-validated-signatures
+export const getPreValidatedSignatures = (from: string): string => {
+  return `0x000000000000000000000000${from.replace(
+    EMPTY_DATA,
+    '',
+  )}000000000000000000000000000000000000000000000000000000000000000001`
+}
+
+// @todo (agustin) check if this can be replaced by shouldExecuteTransaction() and refactor
+const checkIfTxIsExecution = (threshold: number, preApprovingOwner?: string, transaction?: Transaction): boolean =>
+  transaction?.confirmations.size === threshold || !!preApprovingOwner || threshold === 1
+
+const estimateTxGas = async (
   safeAddress: string,
   to: string,
   data: string,
   tx?: Transaction,
   preApprovingOwner?: string,
 ): Promise<number> => {
-  try {
```

Comment:
is it ok to remove try/catch block?

Reply:
Yes because I added it on the wrapper function `estimateTxGasCosts`